slam-ws/src/motor_control/launch/motor_control.launch

```diff
@@ -1,7 +1,7 @@
 <launch>
 
   <!-- Ports -->
-	<arg name="motor_port" default="/dev/ttyACM0"/>
+    <arg name="motor_port" default="/dev/ttyACM0"/>
   <arg name="encoder1_port" default="/dev/ttyACM1"/>
   <arg name="encoder2_port" default="/dev/ttyACM2"/>
```

Comment:
why did the indent change?

Reply:
I switched all the tabs in all our files to 4 spaces